modules/extensions/pom.xml

```diff
@@ -327,6 +329,11 @@
             <classifier>runtime</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.json.wso2</groupId>
+            <artifactId>json</artifactId>
+            <version>${org.wso2.json.version}</version>
```

Comment:
need to move to dependencymanagement

Reply:
fixed via https://github.com/wso2/wso2-synapse/pull/1899/commits/d03967ca95a513fc1c5d958858bfdd887bcb3014